docs/pages/repo/docs/ci/circleci.mdx

```diff
@@ -31,6 +31,7 @@ And a `turbo.json`:
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "build": {
+      "outputs": [".next/**", ".svelte-kit/**"],
```

Comment:
```suggestion
      "outputs": [".next/**", "!.next/cache/**", ".svelte-kit/**"],
```

Reply:
with Workspace Overrides, it's rarely going to make sense to have both next and sveltekit outputs in the same config. The docs for workspace configs even have this exact example. If it isn't necessary, just use one or the other?